src/UKHO.S100PermitService.Common/Validations/ChecksumValidation.cs

```diff
@@ -0,0 +1,23 @@
+﻿using ICSharpCode.SharpZipLib.Checksum;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+
+namespace UKHO.S100PermitService.Common.Validations
+{
+    [ExcludeFromCodeCoverage]
+    public static class ChecksumValidation
+    {
+        private const int EncryptedHardwareIdLength = 32;
+        private const int ReverseChecksumIndex = 6;
```

Comment:
add a line spacing after declaration of variables

Reply:
Done